src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random RANDOM = new Random();
+
+    public String getRandomColor() {
+        Color[] colors = Color.values();
+        return colors[RANDOM.nextInt(colors.length)].toString();
```

Comment:
[Use name() for getting String representation of enum constants](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants)

Reply:
not fixed